socket.io-client/socket.io-client.d.ts

```diff
@@ -1,46 +1,656 @@
-// Type definitions for socket.io-client 1.2.0
+// Type definitions for socket.io-client 1.3.5
 // Project: http://socket.io/
-// Definitions by: PROGRE <https://github.com/progre/>
+// Definitions by: divillysausages <https://github.com/divillysausages/>
```

Comment:
yep. author is reviewer and 1st class user.
please add your name below progre's name.
like this. https://github.com/borisyankov/DefinitelyTyped/blob/master/jquery/jquery.d.ts#L3